packages/devtools_app/lib/src/shared/new_issue.dart

```diff
@@ -9,8 +9,8 @@ import 'utils.dart';
 const newDevToolsIssueUriDisplay = 'github.com/flutter/devtools/issues/new';
 
 Uri newDevToolsIssueUri({String? issueDetails}) {
```

Comment:
now that there is only one use of this `newDevToolsIssueUri` method, it might make make the code easier to follow to put it back in the `issueTrackerLink` method in `extensions_external.dart`. 

Perhaps a greater reason to do this: the code in `newDevToolsIssueUri` is specific to the external extension points, so it would be safer to have **only** the `extensions_external.dart` file have access to this helper. Otherwise, the internal extensions points file will import `new_issue.dart` and will have access to code that was only intended to be used in the external extension points.

 This will also make the code more parallel with how the g3 implementation of extension points is written.